power-pages-docs/admin/admin-center-analytics.md

```diff
@@ -56,13 +56,13 @@ Within the selected environment, you can also select a specific site for which y
 
 ### Considerations
 
-- The charts show DAU, WAU and MAU numbers for the following categories of users:
-    - Authenticated Users shows the unique authenticated users who accessed the websites.
-    - Anonymous Users shows the unique anonymous users who accessed the websites.
-    - All Users shows the unique users who accessed the websites (either by authentication or anonymously).
-- All the traffic numbers (DAU, WAU and MAU) shown are with the intent of understanding the visits to the websites. 
-- The authenticated user counts may vary slightly from license consumption reports. For instance, in cases where the authenticated user already has a Power Apps license, the visit isn't counted in license consumption, but is counted in the total authenticated users visiting the website.
-- The usage numbers can be under reported for the first few days after enabling the feature. For instance, the monthly active users will be accurate after 30 days of this feature being live, and weekly active users will be accurate after seven days of this feature being live.
+- The charts show DAU, WAU, and MAU numbers for the following categories of users:
+    - Authenticated Users show the unique authenticated users who accessed the websites.
+    - Anonymous Users show the unique anonymous users who accessed the websites.
+    - All Users show the unique users who accessed the websites (either by authentication or anonymously).
+- All the traffic numbers (DAU, WAU, and MAU) shown are with the intent of understanding the visits to the websites. 
+- The authenticated user counts may vary slightly from license consumption reports. For instance, in cases where the authenticated user already has a Power Apps license, the visit isn't counted in license consumption but is counted in the total authenticated users visiting the website.
+- The usage numbers can be underreported for the first few days after enabling the feature. For instance, the monthly active users will be accurate after 30 days of this feature being live, and weekly active users will be accurate after seven days of this feature being live.
```

Comment:
- Since that double "r" makes the word look weird, I'd hyphenate "under-reported" or rewrite the sentence to avoid it altogether.
- Same note about "enable" and "disable" as in the first PR I reviewed.
- The second sentence has an awkward verb construction ("being live"). I'd rewrite like this:

For instance, MAU is accurate after 30 days and WAU is accurate after 7 days.

(But I'd also leave an editor's note asking when DAU is accurate.)